extension/intents/tabs/tabs.toml

```diff
@@ -289,6 +292,10 @@ match = """
 [[tabs.collectMentionedTabs.example]]
 phrase = "Collect these tabs"
 
+[[tabs.collectMentionedTabs.example]]
+phrase = "Collect all Google Docs tabs"
+test = true
```

Comment:
Oops, I think you added this to the wrong one – this is a good example to show, but "Undo last closed tab" could be marked as a test

Reply:
Sorry about that, I will remove it and mark "Undo last closed tab" as a test. Thanks!